### PR TITLE
Deleted Postings Sync

### PR DESCRIPTION
**Pull request by lukasbischof**

### CODE REVIEW

File: app/services/job_posting_api/current_job_postings_poller.rb [hunk 1]
```diff
@@ -18,7 +18,7 @@ def initialize
     end
 
     def perform
-      URI.parse(ENV['JOB_POSTINGS_FEED_URL']).open do |page|
+      URI.parse("#{ENV['JOB_POSTINGS_API_URL']}/dev/xml.rss").open do |page|
```
[the remark this answers is not in the document]
Comment (TheFehr) [reply app/services/job_posting_api/current_job_postings_poller.rb:21]: Routes are now:
/current_postings.xml (/dev/current_postings.xml)
and
/deleted_postings.json

[the remark this answers is not in the document]
Comment (TheFehr) [reply app/services/job_posting_api/current_job_postings_poller.rb:21]: also the address now contains a lat and lng

File: .gitpod.Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FROM gitpod/workspace-postgres
```
Comment (TheFehr) [inline .gitpod.Dockerfile:1]: We should probably delete this file

File: .ruby-version [hunk 3]
```diff
@@ -1 +1 @@
-2.7.1
```
Comment (TheFehr) [inline .ruby-version:1]: why?

Comment (lukasbischof) [reply .ruby-version:1]: Because it's the latest patch version. But yes, not really related, I know 😅

File: spec/rails_helper.rb [hunk 4]
```diff
@@ -79,6 +79,11 @@
   config.hook_into :webmock
   config.configure_rspec_metadata!
   config.default_cassette_options = { record: :new_episodes } unless ENV['CI']
+
+  File.read(Rails.root.join('.env.example')).scan(/#?\s*(\S+)=\S*/).flatten.each do |key|
+    config.filter_sensitive_data("[#{key}]") { ENV[key] } if key.match?(/secret|key|password/i)
+  end
+
```
Comment (TheFehr) [inline spec/rails_helper.rb:86]: Why?

**Review by TheFehr — APPROVED** [review]

**Pull request merged**